yt_dlp/extractor/hotstar.py

```diff
@@ -235,3 +245,41 @@ def _real_extract(self, url):
             if video.get('contentId')]
 
         return self.playlist_result(entries, playlist_id)
+
+
+class HotStarSeriesIE(HotStarBaseIE):
+    IE_NAME = 'hotstar:series'
+    _VALID_URL = r'(?:https?://)(?:www\.)?hotstar\.com(?:/in)?/tv/[^/]+/(?P<id>\d{10})$'
+    _TESTS = [{
+        'url': 'https://www.hotstar.com/in/tv/radhakrishn/1260000646',
+        'info_dict': {
+            'id': '1260000646',
+        },
+        'playlist_mincount': 690,
+    }, {
+        'url': 'https://www.hotstar.com/tv/dancee-/1260050431',
+        'info_dict': {
+            'id': '1260050431',
+        },
+        'playlist_mincount': 43,
+    }]
+
+    def _real_extract(self, url):
+        series_id = self._match_id(url)
+        headers = {
+            'x-country-code': 'IN',
+            'x-platform-code': 'PCTV',
+        }
+        detail_json = self._download_json('https://api.hotstar.com/o/v1/show/detail?contentId=' + series_id,
```

Comment:
why not use `self._call_api`?

Reply:
yup
it seems to be for direct video files only and not for series. I tried looking for an endpoint that could be used with `call_api` but couldn't find any.